github/data_source_github_team_repository_test.go

```diff
@@ -0,0 +1,77 @@
+package github
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccGithubTeamRepositorty(t *testing.T) {
```

Comment:
```suggestion
func TestAccGithubTeamRepository(t *testing.T) {
```

Reply:
I'll change it to TestAccGithubTeamRepositories to avoid duplication with test resource_github_team_repository_test.go